src/Delivery/Client.php

```diff
@@ -41,17 +41,25 @@ class Client extends BaseClient
     private $preview;
 
     /**
+     * @var string|null
+     */
+    private $defaultLocale;
+
+    /**
      * Client constructor.
      *
-     * @param string                $token   Delivery API Access Token for the space used with this Client
-     * @param string                $spaceId ID of the space used with this Client.
-     * @param bool                  $preview True to use the Preview API.
+     * @param string                $token         Delivery API Access Token for the space used with this Client
+     * @param string                $spaceId       ID of the space used with this Client.
+     * @param bool                  $preview       True to use the Preview API.
      * @param LoggerInterface       $logger
      * @param GuzzleClientInterface $guzzle
+     * @param string|null           $defaultLocale The default is to fetch the Space's default locale. Set to a locale
+     *                                             string, e.g. "en-US" to fetch content in that locale. Set it to "*"
+     *                                             to fetch content in all locales.
      *
      * @api
      */
-    public function __construct($token, $spaceId, $preview = false, LoggerInterface $logger = null, GuzzleClientInterface $guzzle = null)
+    public function __construct($token, $spaceId, $preview = false, LoggerInterface $logger = null, GuzzleClientInterface $guzzle = null, $defaultLocale = null)
```

Comment:
`null` means it's going to use the Space's default locale. (As configured in Contentful)